### PR TITLE
Maryia/Feat: Config for branding removal

### PR DESCRIPTION
**Pull request by maryia-deriv**

### CODE REVIEW

**Review by matin-deriv — DISMISSED** [review]

File: src/templates/_common/brand.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        dmt5: {
+            name     : 'DMT5',
+            full_name: 'Deriv MetaTrader 5',
+            icon     : 'ic-brand-dmt5.svg',
+        },
+        dxtrade: {
+            name: 'Deriv X',
+            icon: 'ic-brand-dxtrade.svg',
+        },
+        binarybot: {
```
Comment (akmal-deriv) [inline src/templates/_common/brand.config.js]: I would advise not to use our brand name as a key inside the config. It makes less sense if someone decides to fork our repo and still use `binarybot` as a key, so we better make it as generic as possible

Comment (maryia-deriv) [reply src/templates/_common/brand.config.js]: I've renamed it to bbot 👌🏻

Comment (matin-deriv) [reply src/templates/_common/brand.config.js]: @maryia-binary did you update it in other pages?

Comment (maryia-deriv) [reply src/templates/_common/brand.config.js]: @matin-binary just now 😊

**Review by akmal-deriv — APPROVED** [review]
LGTM

**Review by matin-deriv — APPROVED** [review]

**Pull request merged**